ocw-course-v2/ocw-studio.yaml

```
@@ -109,6 +110,68 @@ collections:
           filter_type: "equals"
           value: "Video"
 
+  - category: Content
+    name: link
+    label: Links
+    folder: content/links
```

Does this mean that when we download the course there will be a folder of links?

```suggestion
    folder: content/external-resources
```

---

> Does this mean that when we download the course there will be a folder of links?

Yes.

---

To prevent rendering of content in Hugo's output, you can inject this into the front matter:

```yaml
_build:
  list: True
  render: False
```

`list` controls whether or not the content shows up during templating when iterating things like `site.Pages`, and `render` controls whether or not HTML (or any custom output formats) are generated during the build. This is a content level setting, so you can't control different behavior between online and offline themes but I don't think we need to render "resource pages" for links anyway, we just need the metadata.